lib/mumukit/flow/adaptive_item.rb

```diff
@@ -57,16 +57,16 @@ def pending_siblings?
       pending_siblings.present?
     end
 
-    def passed_siblings_by(submitter)
-      structural_parent.exercise_assignments_for(submitter).select(&:passed?)
+    def solved_siblings_by(submitter)
+      structural_parent.exercise_assignments_for(submitter).select(&:solved?)
     end
 
     def similar_easy_siblings_for_every_tag?
       next_item.tags.all? { |tag| easy_siblings_with(tag).count >= min_easy_siblings_for_skipping }
     end
 
     def easy_siblings_with(tag)
-      passed_siblings_by(submitter).select { |sibling| sibling.easy? && sibling.item.tagged_as?(tag) }
+      solved_siblings_by(submitter).select { |sibling| sibling.easy? && sibling.item.tagged_as?(tag) }
```

Comment:
I'm just worried about this. For this one shouldn't we consider only passed ones?
I'm pretty sure it hasn't an impact because of how it works, but semantically it's confusing. 

I don't have a strong opinion here though, if you think this is good enough, let's go with it

Reply:
There's a catch though: if we _only_ look for passed ones, then it might suggest a skipped one as the next item, as it isn't... well... passed. That's why `solved?` is necessary. We need items not green.

I agreed with you originally but a quick change revealed a few tests failing - it stopped skipping where it should have in a couple cases.

I don't have a strong opinion either, but I can't change if it I want it to work :stuck_out_tongue: so I'm merging.